### ISSUE TITLE
Cannot read property 'each' of undefined

### ISSUE THREAD

**Issue by strarsis** [csstree/csstree#36]
When using css-tree walkDeclarations with a csso AST, a TypeError is thrown.

Sample code for reproducing this issue:
```js
var csso = require('csso'),
    walkDeclarations = require('css-tree').walkDeclarations;

var css = '.test { color: red; }';
var ast = csso.parse(css);

walkDeclarations(ast, function(node, item, list) {
  console.log(node, item);
});
````
Dependencies for this sample:
````
$ npm install csso csstree/csstree
````

css-tree is actually used by csso internally, also for parsing, so there shouldn't be compatibility issues.

**Comment by lahmatiy** [comment]
Last published `csso` version (2.3.1) doesn't use `css-tree`, next version (`3.0`) will do. Therefore csso before `3.0` is incompatible with css-tree

**Comment by strarsis** [comment]
Thanks!
